pkg/e2e/networks_test.go

```diff
@@ -29,47 +29,34 @@ import (
 
 func TestNetworks(t *testing.T) {
 	// fixture is shared with TestNetworkModes and is not safe to run concurrently
-	c := NewCLI(t)
-
 	const projectName = "network-e2e"
+	c := NewCLI(t, WithEnv(
```

Comment:
fyi this test is functionally the same, I just removed the "subtests" since it's really one test, used `WithEnv` to shorten the commands, and added a `-t0` on the `down` so that it's faster